pkg/components/validate_headers.go

```diff
@@ -53,6 +64,7 @@ func (r *validateHeaderTransport) RoundTrip(req *http.Request) (*http.Response,
 // ValidateHeaderConfig is used to validate a map of headers and their allowed values against an incoming requests headers
 type ValidateHeaderConfig struct {
 	Allowed map[string][]string `description:"Map of headers to validate and the allowed values for those headers."`
+	Split   map[string]string   `description:"Map of headers to split on the associated value"`
```

Comment:
Suggestion: tweak the wording on the `Split` description to `Map of headers to the delimiter character to use for headers that contain multiple values.`